plugins/jira/plugin.spec.yaml

```diff
@@ -7,7 +7,8 @@ vendor: rapid7
 support: rapid7
 status: []
 description: Automate the creation, search and management of issues, users, and alerting for Jira Software, Jira Server, and Jira ServiceDesk
-version: 6.1.0
+version: 6.1.1
+supported_versions: ["6.0"]
```

Comment:
Add the versions from here https://github.com/rapid7/insightconnect-plugins/pull/990/files#r707585225

Reply:
```suggestion
supported_versions: ["Jira Server 6.0", "Jira (Cloud)", "Jira ServiceDesk (Cloud)"]
```